packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -501,9 +501,9 @@ export const TooltipPlugin2 = ({
       }
     });
 
-    const onscroll = () => {
+    const onscroll = (e: Event) => {
       updatePlotVisible();
-      _isHovering && !_isPinned && dismiss();
+      _isHovering && !_isPinned && e.target?.contains(_plot!.root) && dismiss();
```

Comment:
this improves tooltip hiding behavior by only hiding when an ancestor is scrolled, and not layout-unrelated panels.